pkg/registry/core/pod/storage/storage.go

```diff
@@ -191,6 +191,10 @@ func (r *BindingREST) Create(ctx context.Context, name string, obj runtime.Objec
 	return
 }
 
+func (r *BindingREST) PreserveObjectMetaSystemFields() bool {
```

Comment:
add a type-assertion like we do for NamedCreater above to make sure we're implementing the interface we think we are

Reply:
and add a comment that we preserve system metadata because the uid in the binding metadata is used as a precondition to update the pod, not to persist (and the timestamps are ignored)

Reply:
Done